src/gui/accountsettings.cpp

```diff
@@ -403,6 +426,16 @@ void AccountSettings::slotOpenCurrentFolder()
     }
 }
 
+void AccountSettings::slotOpenCurrentLocalSubFolder()
+{
+    QModelIndex selected = ui->_folderList->selectionModel()->currentIndex();
```

Comment:
I'm slightly worried that the selected item could be different from the one that was clicked on. I couldn't make that happen in my tests though.